modules/web/sites/README.md

```diff
@@ -536,6 +539,12 @@ module sites './web/sites/main.bicep' = {
     diagnosticStorageAccountId: '<diagnosticStorageAccountId>'
     diagnosticWorkspaceId: '<diagnosticWorkspaceId>'
     enableDefaultTelemetry: '<enableDefaultTelemetry>'
+    hybridConnectionRelays: [
+      {
+        resourceId: '<resourceId>'
```

Comment:
I think this will go away once the readme is regenerated

Reply:
I'm not following why this hybridConnectionRelays block needs to go away?